File: 프로그래머스/unrated/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    var answer = 0;
    
    let x = 1
    let y = 0
    
    num_list.forEach(e => {
        x *= e
        y += e
    })
    
    if (x < y**2) {
        answer = 1
    }
    
    return answer;
}

File: 프로그래머스/unrated/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int times = 1;
        int plus = 0;
        
        for (int num: num_list) {
            times *= num;
            plus += num;
        }
        
        System.out.println(times + " " + plus);
        
        if (times < plus * plus) {
            answer = 1;
        }
            
        return answer;
    }
}